---
event_type: IssuesEvent
avatar: "https://avatars.githubusercontent.com/u/44277022?"
user: jwhite242
date: 2024-03-05
repo_name: flux-framework/flux-core
html_url: https://github.com/flux-framework/flux-core/issues/5770
repo_url: https://github.com/flux-framework/flux-core
---

<a href='https://github.com/jwhite242' target='_blank'>jwhite242</a> open issue <a href='https://github.com/flux-framework/flux-core/issues/5770' target='_blank'>flux-framework/flux-core#5770</a>.

<p>Standalone python bindings out of sync with flux core ~version 0.57+</p><small>There's some discussion of this already here: https://github.com/flux-framework/flux-python/issues/9 .  From the comments there it seems the callbacks are out of sync. Believe it's mostly an...</small><a href='https://github.com/flux-framework/flux-core/issues/5770' target='_blank'>View Comment</a>